packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -780,24 +780,39 @@ export class AppStudioPluginImpl {
         AppStudioError.NotADirectoryError.message(appDirectory)
       );
     }
+    const zip = new AdmZip();
+    zip.addFile(Constants.MANIFEST_FILE, Buffer.from(manifestString));
+
     const manifest: TeamsAppManifest = JSON.parse(manifestString);
-    const colorFile = `${appDirectory}/${manifest.icons.color}`;
 
-    let fileExists = await this.checkFileExist(colorFile);
-    if (!fileExists) {
-      throw AppStudioResultFactory.UserError(
-        AppStudioError.FileNotFoundError.name,
-        AppStudioError.FileNotFoundError.message(colorFile)
-      );
+    // color icon
+    if (manifest.icons.color && !manifest.icons.color.startsWith("https://")) {
```

Comment:
do we need to consider the case that the url of icon starts from `http://`?

Reply:
Need more investigation to decide if we need to support this case. Refer to VSO 12681229.